### ISSUE TITLE
How can I feed batch of data to the model when predicting?

### ISSUE THREAD

**Issue by zhimakaimenxa** [migueldeicaza/TensorFlowSharp#490]
My code looks like follows,,

                var runner = session.GetRunner();
                //var tensor = Utils.ImageToTensorGrayScale(file);
                var tensor = dataTransfer.OutputTensor(symbol, beginIndex, endIndex);
                runner.AddInput(graph["input_1"][0], tensor);
                runner.Fetch(graph["dense_2/Softmax"][0]);

                var output = runner.Run();
                var vecResults = output[0].GetValue();

The batch size dimension of the tensor can only be 1, when I give tensor with multi samples, the result are same. How can I predict a batch of samples in a time without using loop?

Best Regards

zhimakaimenxa

**Comment by cesarsouza** [comment]
While I kind of love this project and where it came from, I would highly suggest you to do this kind of processing in Python in either Python/TensorFlow or Python/Pytorch and simply consume their outputs in a C# application if you need to.
